assets/texts/fr/amoSummary.txt

```diff
@@ -1,2 +1,3 @@
-Ce module complémentaire vous donne un bon petit sélecteur d'émoticônes que vous pouvez pour en trouver 🔎 et en sélectionner. 😀
-Il sera ajouté à votre presse-papier 📋 (de façon similaire à un "copier-coller") quand vous cliquez 🖱️ dessus.
```

Comment:
"de façon similaire" has been removed for the "copy of the copy-paste", so it might be problematic; must see if I can't change that without destroying the char limit.

Reply:
Fixed as of bba933bd